From: Randall Vasquez<ran.dall@icloud.com>

Fixes the version output

Signed-off-by: Randall Vasquez <ran-dall@icloud.com>

diff -Naur a/lib/git-subrepo b/lib/git-subrepo
--- a/lib/git-subrepo	2021-10-28 02:03:51.505741029 -0700
+++ b/lib/git-subrepo	2021-10-28 12:45:05.001488328 -0700
@@ -30,7 +30,7 @@
 bash+:import :std can version-check
 
 
-VERSION=0.4.3
+VERSION=0.4.5
 REQUIRED_BASH_VERSION=4.0
 REQUIRED_GIT_VERSION=2.7.0
 GIT_TMP=$(git rev-parse --git-common-dir 2> /dev/null || echo .git)/tmp
@@ -1657,8 +1657,8 @@
     # If `git-subrepo` was system-installed (`make install`):
     if [[ -e $lib/git-subrepo.d/upstream ]] &&
        [[ -e $lib/git-subrepo.d/commit ]]; then
-      command_remote=$(readlink "$lib/git-subrepo.d/upstream")
-      command_commit=$(readlink "$lib/git-subrepo.d/commit")
+      command_remote=$(bash "$lib/git-subrepo.d/upstream")
+      command_commit=$(bash "$lib/git-subrepo.d/commit")
     elif [[ $lib =~ / ]]; then
       lib=$(dirname "$lib")
       if [[ -d $lib/.git ]]; then
diff -Naur a/lib/git-subrepo.d/commit b/lib/git-subrepo.d/commit
--- a/lib/git-subrepo.d/commit	1969-12-31 16:00:00.000000000 -0800
+++ b/lib/git-subrepo.d/commit	2021-10-28 12:35:07.719564728 -0700
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo $(GIT_DIR=.git git rev-parse --short HEAD)
diff -Naur a/lib/git-subrepo.d/upstream b/lib/git-subrepo.d/upstream
--- a/lib/git-subrepo.d/upstream	1969-12-31 16:00:00.000000000 -0800
+++ b/lib/git-subrepo.d/upstream	2021-10-28 12:35:48.239559545 -0700
@@ -0,0 +1,2 @@
+#! /bin/sh
+echo "https://github.com/ingydotnet/git-subrepo.git"
